templates/_helpers.tpl

```diff
@@ -29,4 +29,11 @@ Create imagepullsecretfor docker registry
 {{- with .Values.imageCredentials }}
 {{- printf "{\"auths\":{\"ghcr.io\":{\"username\":\"cobrowse-enterprise\",\"password\":\"%s\",\"auth\":\"%s\"}}}" .password (printf "cobrowse-enterprise:%s" .password | b64enc) | b64enc }}
 {{- end }}
-{{- end }}
\ No newline at end of file
+{{- end }}
+
+{{/*
+Create the default SecurityContext for all service components
+*/}}
+{{- define "defaultSecurityContext" }}
+{{- dict "allowPrivilegeEscalation" false "privileged" false "readOnlyRootFilesystem" true | toYaml }}
```

Comment:
We don't have to add `runAsNonRoot: true` so long as we've specified `runAsUser` to a non-0 value, which we are doing in the pod templates.